### ISSUE TITLE
error: package com.RNpublic.neuraintegration does not exist

### ISSUE THREAD

**Issue by samzmann** [NeuraLabs/react-native-neura#1]
I am following the install steps for Android and run onto this build error pretty quickly:

```
/Users/sam/pickpack/uidev/android/app/src/main/java/com/pkup/MainApplication.java:39: error: package com.RNpublic.neuraintegration does not exist
import com.RNpublic.neuraintegration.NeuraIntegrationSingleton;
```

Could it be that there are additional install steps not listed in the README?

**Comment by malren** [comment]
@LaVielle what version of React are you using?

**Comment by samzmann** [comment]
@malren 
I am using:
react-native 0.59.9
react-native-neura 1.0.6

I got the build to succeed by adding a few things in settings.gradle and some other files
I will update my comment with solution steps once I am sure my integration is working (not just building)

**Comment by samzmann** [comment]
Somehow it is not possible to access the object when mounting it to react-native android. 

` TypeError: Cannot read property 'authenticateAnon' of undefined`

Can you please upload a sample project to see exact implementation details.

**Comment by malren** [comment]
@LaVielle 
From our experience, when using react-native version < 0.6, manual linking is needed (might also explain the initial issue you've experienced, as their's no need to add anything to the settings.gradle file):

**Android**:
1. Move the react-native-neura iOS folder (from your node_modules directory) to a different location 
2. run `react-native link react-native-neura`
3. Return the iOS folder back.

**iOS**:  
Add the following to your pod file:
`pod 'react-native-neura', :podspec => '../node_modules/react-native-neura/react-native-neura.podspec'`

Let me know if this solved the issue.

**Comment by samzmann** [comment]
Ok we got it working. It was quite hard to debug. What really helped in the end was the logcat native logs:
`adb logcat | grep -F "Neura"`

I would add that inking is required to the documentation. We tried that before and I am sure this is a very unique issue for us why it did not work but I share it anyways. We are combining braze and neura and in braze they have a unique `settings.gradle` setup that we accidentally copied over.

So we had `settingsDear` rather than the correct `rootProject.projectDir` in the new File command

This is working:
```
include ':react-native-neura'
project(':react-native-neura').projectDir = new File(rootProject.projectDir, '../node_modules/react-native-neura/android')
include ':react-native-appboy-sdk'
project(':react-native-appboy-sdk').projectDir = new File(settingsDir, '../node_modules/react-native-appboy-sdk/android')
```

Is it somehow possible now to debug if the background push function is properly setup from the dashboard or the app?
